### ISSUE TITLE
Fix a11y issues

### ISSUE THREAD

**Issue by darkwebdev** [eBay/ebayui-core-react#167]
1. EbayMenu:
<img width="1267" alt="image" src="https://user-images.githubusercontent.com/2737310/203552203-df4d5055-f39d-450a-8ce3-5b69300b887e.png">

2. EbayIconButton should have `aria-label` required.

**Comment by darkwebdev** [comment]
possible solution for IconButton:
<img width="316" alt="image" src="https://user-images.githubusercontent.com/2737310/204255387-28f3a75f-9d55-4250-8c14-027e16cef67e.png">
and in FakeMenuButton:
<img width="616" alt="image" src="https://user-images.githubusercontent.com/2737310/204255599-eb90ec67-9765-4057-ad83-8fdf5b28b2e1.png">
